lib/utilities/stringChecks.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * hasNonPrintables returns whether or not the given value
+ * includes characters that are non-printable.
+ *
+ * @param {string} value - value to check for non-printables
+ * @returns {Boolean} whether or not the value has non-printables
+ */
+function hasNonPrintables(value) {
+    for (const char of value) {
+        const codePoint = char.codePointAt(0);
+        if (codePoint < 32) {
```

Comment:
127 is also nonprintable:

```suggestion
        if (codePoint < 32 || codePoint === 127) {
```